src/routes/LPOptimisers/VaultFormRoute/VaultWithdrawRolloverForm/VaultWithdrawRolloverForm.tsx

```diff
@@ -37,12 +41,25 @@ export const VaultWithdrawRolloverForm: React.FunctionComponent<VaultWithdrawRol
   const weights = distribution === 'automatic' ? automaticWeights : manualWeights;
   const combinedWeightValue = weights.reduce((total, weight) => total + weight.distribution, 0);
 
+  const spareWeights = weights
```

Comment:
lets extract utility from this. Maybe inside helpers?

Reply:
dc6a8d30ed67af8aba91cf3ea2a8fcdd1bb73dab